src/Files.App/Views/SettingsPages/Tags.xaml.cs

```diff
@@ -121,7 +122,7 @@ private bool IsNameValid(string name)
 				string.IsNullOrWhiteSpace(name) ||
 				name.StartsWith('.') ||
 				name.EndsWith('.') ||
-				(name != editingTag?.Tag.Name && ViewModel.Tags.Any(tag => name == tag.Tag.Name))
+				ViewModel.Tags.Any(tag => name == tag.Tag.Name)
 			);
```

Comment:
Can you revert this?
At the moment the user can't save the changes if he has only changed the color

Reply:
I haven't reverted it but it should work as intended now